org.flathub.p3x-onenote.yml

```diff
@@ -0,0 +1,77 @@
+app-id: org.flathub.p3x-onenote
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - "--socket=wayland"
+  - "--share=ipc"
+  - "--device=dri"
+  - "--socket=pulseaudio"
+  - "--filesystem=home"
```

Comment:
why is this needed?

Reply:
Where does it store configs? It's supposed to follow, XDG standards.

Also why pulseaudio? Seems like a note taking app

Reply:
the readme says it uses userdata path https://www.npmjs.com/package/electron-store means it respects `$XDG_CONFIG_HOME`, inside the flatpak XDG_CONFIG_HOME is defined to be under `~/.var/app/app-id/config` - so it should work without `home` access.

Reply:
The point here still stands.